packages/modal/src/Modal.tsx

```diff
@@ -0,0 +1,268 @@
+import React, { Fragment, useCallback, SetStateAction } from 'react';
+import PropTypes from 'prop-types';
+import Portal from '@leafygreen-ui/portal';
+import Icon, { Size } from '@leafygreen-ui/icon';
+import { useEventListener } from '@leafygreen-ui/hooks';
+import { uiColors } from '@leafygreen-ui/palette';
+import { css, cx } from '@leafygreen-ui/emotion';
+
+export const ModalSize = {
+  XXSmall: 'xxsmall',
+  XSmall: 'xsmall',
+  Small: 'small',
+  Default: 'default',
+  Large: 'large',
+  XLarge: 'xlarge',
+} as const;
+
+export type ModalSize = typeof ModalSize[keyof typeof ModalSize];
+
+const overlayStyle = css`
+  animation: fade-in 500ms cubic-bezier(0.165, 0.84, 0.44, 1);
+  -webkit-animation: fade-in 500ms cubic-bezier(0.165, 0.84, 0.44, 1);
+  background-color: ${uiColors.black};
+  opacity: 0.6;
+  position: fixed;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  overflow: hidden;
+`;
+
+const modalContentStyle = css`
+  margin: 18px 40px;
+  max-width: 1270px;
+  min-width: 944px;
+  padding: 30px;
+  color: ${uiColors.gray.dark3};
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  background-color: ${uiColors.white};
+  border: 1px solid ${uiColors.gray.light3};
+  border-radius: 3px;
+  box-shadow: 0 5px 15px rgba(0, 0, 0, 0.3);
+  margin-right: auto;
+  margin-left: auto;
```

Comment:
We declare a different margin on the first line of this style declaration. We should remove these two rules, and update the first margin declaration to: `margin: 18px auto;`

Reply:
👍